src/css-rule-set-to-string.js

```diff
@@ -5,9 +5,14 @@ import camelCasePropsToDashCase from './camel-case-props-to-dash-case';
 import mapObject from './map-object';
 import {getPrefixedStyle} from './prefixer';
 
+function quoteValueIfNeeded(property, value) {
+  const quotedProperties = ['content'];
```

Comment:
Once this is pulled into its own file, this array can be defined and initialized outside of the function. That will prevent us from creating lots of array.